### DISCUSSION TITLE
OpenDTU - keine Verbindung zum WR mehr nach >1 Jahr einwandfreier Funktion

### DISCUSSION THREAD

**Discussion by Hamstihamsta** [tbnobody/OpenDTU#2224] [Q&A]
Hallo zusammen, 

seit einiger Zeit scheint keine Verbindung zwischen meiner OpenDTU und meinem HM-800 Wechselrichter mehr zu bestehen. Zuvor hat es zuverlässig über ein Jahr lang funktioniert. Der Wechselrichter blinkt grün und speist auch tatsächlich ein. 

![grafik](https://github.com/user-attachments/assets/702dce5f-e197-402e-9371-df18d1274a2a)

An manchen Tagen wurde wohl auch der Ertrag mehrerer Tage an einem späteren Tag geloggt, zum Beispiel am 16.08.

Es gab aber keine Änderungen, weder an der Hardware noch an der Software. Was ich diese Woche gemacht habe:
Verkabelung der OpenDTU getauscht
Firmwareupdate
Sendeleistung erhöht
OpenDTU ca. 1 Meter neben dem Wechselrichter platziert

Jedoch ohne Besserung. 

Systeminformationen:
![grafik](https://github.com/user-attachments/assets/6f1d8e3a-f9c7-4a37-a338-965c719a8e9b)
![grafik](https://github.com/user-attachments/assets/5ed283a0-23fd-4481-bb6f-feed226f8db1)

Konsole:
RX Period End
16:40:25.658 > All missing
16:40:25.658 > Nothing received, resend whole request
16:40:25.658 > TX RealTimeRunData Channel: 23 --> 15 90 11 21 70 80 19 21 96 80 0B 00 66 C9 F0 DB 00 00 00 00 00 00 00 00 EB 10 1F 

Was könnte ich noch tun, um das Problem zu lösen? 

Vielen Dank im Voraus! :-)

**Comment by ghost** [comment]
Hi,

keine Ahnung, gelegentlich tauchen hier immer wieder mal [ähnliche Probleme](https://github.com/tbnobody/OpenDTU/discussions/1167) auf.  Du könntest  versuchen die Frequenz zu ändern. Das kann allerdings etwas dauern, bis der WR es merkt, steht so in der verlinkten Disskussion,
 (Weiteres [ähnliches Thema](https://github.com/tbnobody/OpenDTU/discussions/1300))

**Comment by Solarteur** [comment]
Bei 1 m Abstand OpenDTU - Wechselrichter **muss** das System mit der geringsten  Sendeleistung vernünftig arbeiten. 

Hast du Möglichkeit, ein anderes nRF24-Modul zu testen? 

Es wäre nicht der erste Fall,  dass ein nRF24-Modul auf einmal nicht mehr funktioniert. 

PS: Bei einem HM-xxx-Wechselrichter ist eine Änderung der Frequenz kein Thema (nRF24-Modul)

**Reply by Hamstihamsta** [reply]
Hallo Solarteur, habe leider kein anderes nRF24 Modul. Aber notfalls kauf ich mir dann einfach eins, ich glaub irgendwie nicht, dass das Problem am Wechselrichter liegt und den einzuschicken, wäre nervig. 

Was meinst Du genau mit der Änderung der Frequenz? Ist das beim HM-800 nicht möglich? Innerhalb der OpenDTU habe ich keine Möglichkeit gefunden, die Frequenz zu ändern. 

Vielen Dank!

**Comment by Solarteur** [comment]
Bei Wechselrichtern, die über das CMT2300A-Modul mit OpenDTU verbunden werden (z. B. aus der HMS-Serie), kann man verschiedene Frequenzen auswählen. Beim nRF24-Modul ist das nicht möglich und auch nicht notwendig. Die genauen Hintergründe kenne ich auch nicht.

**Reply by ghost** [reply]
Hi,
ich habe total gepennt, Frequenzeinstellen geht nur beim CMT, danke für den Hinweis.
Das NRF24 Modul kann natürlich hin sein.

**Comment by Hamstihamsta** [comment]
Moin moin, mein Plan war eigentlich: Wenn ich (grad auf Dienstreise) wieder daheim bin, bestelle ich ein neues NRF24 Modul. 

Jetzt geht es aber wieder... gemacht habe ich nichts.

Ich beobachte das mal weiter, vielen Dank für eure Hilfe! :-)
